### ISSUE TITLE
Request for additional information to be added to herbs.

### ISSUE THREAD

**Issue by Owai-Seek** [Asthereon/green-thumb#12]
This is more of a minor compatibility complaint than anything, but it'd be nice if it were resolved.

Basically, I use both Green Thumb, and Herblore Recipes. Herblore Recipes, and because of this, the tooltips for "Uses" both stack.

![image](https://github.com/Asthereon/green-thumb/assets/7543955/e04a5829-75e2-43a2-a9f9-3fb180cd57c6)

My request would simply be to add that additional information to this plugin. Herblore Recipes allows you to disable its tooltips on an individual basis, so I'd be able to disable it on Seeds, and only see the information once, rather than twice. (The only reason I'm not doing that currently, is because that plugin tells me which level I can do things at, which is useful information to have.)

The alternative approach would be the ability to toggle off the "Uses" information based on what kind of seed it is. (Trees, Herbs, Allotment, ect), but I'm fairly certain that would be a much more tedious process.

**TL:DR**
It'd be nice to have the Herblore Level requirements (and secondary ingredients) for potions shown on Herb Seed Use tooltips.
